app/assets/javascripts/discourse/app/controllers/edit-slow-mode.js

```diff
@@ -0,0 +1,109 @@
+import Controller from "@ember/controller";
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+import discourseComputed from "discourse-common/utils/decorators";
+import I18n from "I18n";
+import Topic from "discourse/models/topic";
+import { fromSeconds, toSeconds } from "discourse/helpers/slow-mode";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Controller.extend(ModalFunctionality, {
+  selectedSlowMode: null,
+  hours: null,
+  minutes: null,
+  seconds: null,
+  saveDisabled: false,
+
+  slowModes: [
```

Comment:
this should be moved in init, with slowModes being null, and the array set in the init, otherwise you leak this array